http/file_server.ts

```diff
@@ -620,37 +620,7 @@ export async function serveDir(req: Request, opts: ServeDirOptions = {}) {
 }
 
 function normalizeURL(url: string): string {
-  let normalizedUrl = url;
-
-  try {
-    //allowed per https://www.w3.org/Protocols/rfc2616/rfc2616-sec5.html
-    const absoluteURI = new URL(normalizedUrl);
```

Comment:
because `url` of Request object is already parsed once ( https://github.com/denoland/deno/blob/7a7767262a3b373cc9a3f6260a8ebd45da7edd1c/ext/fetch/26_fetch.js#L370 ), this can't throw an error.